### PR TITLE
CI: Add some helper functions

### PR DESCRIPTION
**Pull request by miska**
Maintainer: @aparcar 
Compile tested: NA
Run tested: NA

Description:
Add some minor helper functions that could be used to produce nicer output during CI runs. How it works can be seen in #13540

### CODE REVIEW

File: .github/workflows/ci_helpers.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# shellcheck disable=SC2039
+color_out() {
+	echo -en "\033[0;$1m"
```
Comment (neheb) [inline .github/workflows/ci_helpers.sh:5]: Why not use printf?

Comment (miska) [reply .github/workflows/ci_helpers.sh]: Out of habit. Replaced with printf.

**Comment by neheb** [comment]
Looks good.

**Comment by aparcar** [comment]
Any reason not to include the helper functions directly in the entrypoint?

**Comment by miska** [comment]
Individual test are executed, not sourced, so the helper functions wouldn't be available. I could change it to sourcing, but then individual test might influence the environment for the following tests.

**Comment by aparcar** [comment]
So individual tests source the CI variable? I guess I'm fine with it

**Pull request merged**